gax-java/gax-grpc/pom.xml

```diff
@@ -63,6 +63,10 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-protobuf</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-s2a</artifactId>
+    </dependency>
```

Comment:
This is the only config change that will be needed. 

The rest are needed for tests to pass locally. Once https://github.com/googleapis/google-auth-library-java/pull/1560 and https://github.com/googleapis/sdk-platform-java/pull/3240 submitted, the config changes in this PR to reference my local snapshots can be removed.

Reply:
fyi - I just merged #3240

Reply:
Thanks Min! In https://github.com/googleapis/sdk-platform-java/pull/3326/commits/8469a1a9b9bf1360646f273ea32a8a65319e1503 I merged those changes in.